backend/benefit/applications/management/commands/seed.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from django.core.management.base import BaseCommand
+
+from applications.enums import ApplicationStatus
+from applications.models import Application, ApplicationBasis
+from applications.tests.factories import ApplicationFactory
+
+
+class Command(BaseCommand):
+    help = "Seed the database with applications"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--number",
+            type=int,
+            default=10,
+            help="Number of applications to create",
+        )
+
+    def handle(self, *args, **options):
+        total_created = len(ApplicationStatus.values) * options["number"]
+        if os.getenv("DEBUG") != "1":
+            self.stdout.write("Seeding is only allowed in debug mode")
```

Comment:
Actually, just out of curiosity, why not just use `print()`? I believe that `stdout.write()` doesn't put the newline at the end of the line (`print()` has an `end` parameter if that's the reason)?

Reply:
No other reason for using stdout.write besides that Django documentation uses stdout.write in their examples for creating management commands. The only practical difference is indeed the newline after calling print(). I guess command line tools could at some scenarios have progress indicators like "#########" that need to be printed on the same line.